clearnode/docs/API.md

```diff
@@ -261,6 +264,25 @@ Currently, `Transfer` supports ledger account of another user as destination (wa
   }], 1619123456789],
   "sig": ["0x9876fedcba..."]
 }
+
+// OR
+
+{
+  "req": [1, "transfer", [{
+    "destination_tag": "UX123D8C", // User tag
```

Comment:
If we decide to go with userTag instead of tag, then this should be changed into something like:

```suggestion
    "destination_usertag": "UX123D8C", // User tag
```